stan/math/prim/scal/prob/pareto_log.hpp

```diff
@@ -1,123 +1,29 @@
 #ifndef STAN_MATH_PRIM_SCAL_PROB_PARETO_LOG_HPP
 #define STAN_MATH_PRIM_SCAL_PROB_PARETO_LOG_HPP
 
-#include <stan/math/prim/scal/meta/is_constant_struct.hpp>
-#include <stan/math/prim/scal/meta/partials_return_type.hpp>
-#include <stan/math/prim/scal/meta/OperandsAndPartials.hpp>
-#include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
-#include <stan/math/prim/scal/err/check_greater_or_equal.hpp>
-#include <stan/math/prim/scal/err/check_nonnegative.hpp>
-#include <stan/math/prim/scal/err/check_not_nan.hpp>
-#include <stan/math/prim/scal/err/check_positive_finite.hpp>
-#include <stan/math/prim/scal/fun/constants.hpp>
-#include <stan/math/prim/scal/fun/multiply_log.hpp>
-#include <stan/math/prim/scal/fun/value_of.hpp>
-#include <stan/math/prim/scal/meta/include_summand.hpp>
-#include <stan/math/prim/scal/meta/contains_nonconstant_struct.hpp>
-#include <stan/math/prim/scal/meta/VectorBuilder.hpp>
-#include <boost/random/exponential_distribution.hpp>
-#include <boost/random/variate_generator.hpp>
-#include <cmath>
+#include <stan/math/prim/scal/prob/pareto_lpdf.hpp>
 
 namespace stan {
   namespace math {
 
-    // Pareto(y|y_m, alpha)  [y > y_m;  y_m > 0;  alpha > 0]
+    /**
+     * @deprecated use <code>pareto_lpdf</code>
```

Comment:
These are all missing argument and template parameter doc, but I suppose that's OK given that it's deprecated.